ci-operator/step-registry/mco-day2/install-kernel-type/OWNERS

```diff
@@ -0,0 +1,15 @@
+approvers:
+  - yunjiang29
+  - jianlinliu
+  - jinyunma
+  - Xia-Zhao-rh
+  - kuiwang02
+  - bandrade
+  - jianzhangbjz
+reviewers:
+  - Xia-Zhao-rh
```

Comment:
since this step has a common root with the multi-arch QE team, do you mind adding me as approver and reviewer?

Reply:
Done